helm/charts/determined/templates/db-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
```

Comment:
blocking: Why aren't we using the [postgres helm chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) here? The current approach will probably work for the time being but anyone seriously operating postgres would likely want to set it up as a StatefulSet and have the ability to scale out if necessary. The official chart will help with this and other gotchas around running Postgres on k8s.

Reply:
Talked about this offline. We are going to document (when we write k8 docs) how to use a separate postgres which is not deployed by us via helm. One of the options for these separate postgres  deployments would be using the postgres helm chart but we won't be managing that. 

Additonally I am going to update this helm chart to make it straightforward to switch between our postgres deployment and their own.